src/mlpack/methods/naive_bayes/nbc_main.cpp

```diff
@@ -77,6 +77,8 @@ PARAM_FLAG("incremental_variance", "The variance of each class will be "
 PARAM_MATRIX_IN("test", "A matrix containing the test set.", "T");
 PARAM_UMATRIX_OUT("output", "The matrix in which the predicted labels for the"
     " test set will be written.", "o");
+PARAM_MATRIX_OUT("output_proba", "The matrix in which the predicted probability of labels for the"
```

Comment:
Go with `output_probs` not `output_proba` please, for consistency across other programs.